pkg/scalers/rabbitmq_scaler.go

```diff
@@ -310,6 +322,10 @@ func parseTrigger(meta *rabbitMQMetadata, config *ScalerConfig) (*rabbitMQMetada
 		meta.mode = rabbitModeQueueLength
 		meta.value = queueLength
 
+		if meta.activationValue > queueLength {
+			return nil, fmt.Errorf("activationValue is greater than value")
+		}
+
```

Comment:
This check it's not necessary (please remove it), the activation value could be higher because it's for the wait before activation the scaler and the queueLenght is for the scaling period.